codegenerator/cli/templates/dynamic/codegen/src/RegisterHandlers.res.hbs

```diff
@@ -22,123 +22,138 @@ let registerContractHandlers = (
   }
 }
 
-// TODO: Start using only config returned by registerAllHandlers instead of Config.getGenerated and Config.setGenerated
 %%private(
-  let chains = [
-    {{#each chain_configs as | chain_config |}}
-    {
-      let contracts = [
-        {{#each chain_config.codegen_contracts as | contract |}}
-        {
-          Config.name: "{{contract.name.capitalized}}",
-          abi: Abis.{{contract.name.uncapitalized}}Abi->Ethers.makeAbi,
-          addresses: [
-            {{#each contract.addresses as | address |}}
-            {{#if is_evm_ecosystem}}
-            "{{address}}"->Address.Evm.fromStringOrThrow
-            {{else}}
-            "{{address}}"->Address.unsafeFromString
-            {{/if}},
-            {{/each}}
-          ],
-          events: [
-            {{#each contract.events as | event |}}
-            module(Types.{{contract.name.capitalized}}.{{event.name.capitalized}}),
-            {{/each}}
-          ],
-        },
-        {{/each}}
-      ]
-      let chain = ChainMap.Chain.makeUnsafe(~chainId={{chain_config.network_config.id}})
-      {{#if chain_config.network_config.rpc_config }}
-      let rpcConfig: Config.rpcConfig = {
-        provider: Ethers.JsonRpcProvider.make(
-          ~rpcUrls={{vec_to_array chain_config.network_config.rpc_config.urls}},
-          ~chainId={{chain_config.network_config.id}},
-          ~fallbackStallTimeout={{chain_config.network_config.rpc_config.sync_config.fallback_stall_timeout}},
-        ),
-        {{#with chain_config.network_config.rpc_config.sync_config as | sync_config |}}
-        syncConfig: Config.getSyncConfig({
-          initialBlockInterval: {{sync_config.initial_block_interval}},
-          backoffMultiplicative: {{sync_config.backoff_multiplicative}},
-          accelerationAdditive: {{sync_config.acceleration_additive}},
-          intervalCeiling: {{sync_config.interval_ceiling}},
-          backoffMillis: {{sync_config.backoff_millis}},
-          queryTimeoutMillis: {{sync_config.query_timeout_millis}},
-        }),
-        {{/with}}
-      }
-      {{/if}}
+  let makeGeneratedConfig = () => {
```

Comment:
I would suggest reviewing this with hide whitespace setting

Reply:
Thanks, I didn't know about the option